app/src/main/res/layout-land/topic_practice_header_view.xml

```diff
@@ -34,12 +32,12 @@
       android:id="@+id/skills_description_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginTop="8dp"
+      android:layout_marginTop="4dp"
       android:fontFamily="sans-serif"
       android:text="@string/topic_practice_skills_description"
       android:textColor="@color/oppiaSecondaryText"
       android:textSize="16sp"
       app:layout_constraintStart_toStartOf="@+id/master_skills_text_view"
       app:layout_constraintTop_toBottomOf="@+id/master_skills_text_view" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done